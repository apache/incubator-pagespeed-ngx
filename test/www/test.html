<html>
  <head>
    <link rel="stylesheet" href="style.css">
    <title>Title</title>
  </head>
  <body>
    <h1>Example Html Page</h1>
    <script src="script.js"></script>
    <script> alert("an alert"); </script>
    <p>
      This is an example.

      
      <!-- a comment -->

      <!-- another comment -->

      <b>bold text</b>

      <img src="Puzzle.jpg">
    </p>
  </body>
</html>


