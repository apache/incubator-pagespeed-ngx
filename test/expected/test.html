<html>
<head>
<link rel=stylesheet href=style.css>
<title>Title</title>
</head>
<body>
<h1>Example Html Page</h1>
<script src=script.js></script>
<script> alert("an alert"); </script>
<p>
This is an example.
<b>bold text</b>
</p>
</body>
</html>
<!-- Processed through ngx_pagespeed using PSOL version 0.10.0.0 -->
